src/pyrokinetics/gk_code/__init__.py

```diff
@@ -6,6 +6,7 @@
 from .gk_input import GKInput, read_gk_input, supported_gk_input_types
 from .gk_output import GKOutput, read_gk_output, supported_gk_output_types
 from .gs2 import GKInputGS2, GKOutputReaderGS2  # noqa
+from .stella import GKInputSTELLA  # , GKOutputReaderSTELLA  # noqa
```

Comment:
Should we add the `OutputReader` too?

Reply:
The output reader probably does not work at the moment. I will try to get a minimal version working this week.